content/bn/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,288 @@
+---
+title: হ্যালো Minikube (Hello Minikube)
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "Get Started"
+    weight: 10
+    post: >
+      <p> কাজ শুরু করতে প্রস্তুত? একটি সাধারণ কুবারনেটিস ক্লাস্টার তৈরি করুন যা একটি নমুনা অ্যাপ চালায় । </p>
+
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+এই টিউটোরিয়ালে দেখবো কিভাবে একটি অ্যাপ্লিকেশনকে মিনিকিউব ও কাটাকোডা ব্যবহার করে কুবারনেটিস ক্লাস্টারে চালানো যাবে।
+
+{{< note >}}
+ইন্সটল করার জন্য
+[minikube স্টার্ট](https://minikube.sigs.k8s.io/docs/start/)  এর নির্দেশনা দেখো।
+{{< /note >}}
+
+## {{% heading "objectives" %}}
+
+* মিনিকিউবে একটি সরল অ্যাপ্লিকেশন স্থাপন করুন।
+* অ্যাপ্লিকেশনটিকে চালান।
+* অ্যাপ্লিকেশন লগ (log) দেখুন।
+
+## {{% heading "prerequisites" %}}
+
+এই টিউটোরিয়ালটি একটি কন্টেইনার ইমেজ প্রদান করে যা NGINX ব্যবহার করে সমস্ত অনুরোধে সাড়া দেয়।
+
+<!-- lessoncontent -->
+
+## একটি মিনিকিউব ক্লাস্টার তৈরি করা।
+
+1. ক্লিক করুন **Launch Terminal** এ
+
+    {{< kat-button >}}
+
+{{< note >}}
+মিনিকুব স্থানীয়ভাবে ইনস্টল করা থাকলে, `minikube start` চালান। `minikube dashboard` কমান্ড কার্যকর করার আগে, একটি নতুন টার্মিনাল খুলুন, সেই টার্মিনালে `minikube dashboard` কমান্ডটি চালান এবং মূল টার্মিনালে ফিরে যান।
+{{< /note >}}
+
+2. একটি ব্রাউজারে কুবারনেটিস ড্যাশবোর্ড খুলুন:
+
+    ```shell
+    minikube dashboard
+    ```
+
+3. Katacoda পরিবেশ (Katacoda Environment): টার্মিনাল প্যানেলের শীর্ষে প্লাস ক্লিক করুন, তারপরে ক্লিক করুন **Select port to view on Host 1** ।
+
+4. Katacoda পরিবেশ (Katacoda Environment): `30000` লিখুন এবং **Display Port** এ ক্লিক করুন।
+
+{{< note >}}
+আপনি যখন  `minikube dashboard` কমান্ড ইস্যু করেন, তখন ড্যাশবোর্ড অ্যাড-অন এবং প্রক্সি সক্রিয় হয় এবং প্রক্সিতে সংযোগ করার জন্য একটি ডিফল্ট ওয়েব ব্রাউজার উইন্ডো খোলে।
+আপনি ড্যাশবোর্ড থেকে কুবারনেটস সংস্থান তৈরি করতে পারেন যেমন স্থাপনা বা পরিষেবা।
```

Comment:
```suggestion
আপনি ড্যাশবোর্ড থেকে কুবারনেটিস সংস্থান তৈরি করতে পারেন যেমন স্থাপনা বা পরিষেবা।
```

Reply:
I can't add these suggestions to a batch, is there any issue?